test/parallel/test-http-server-unconsume-consume.js

```diff
@@ -4,8 +4,8 @@ const http = require('http');
 
 const testServer = http.createServer(common.mustNotCall());
 testServer.on('connect', common.mustCall((req, socket, head) => {
-  socket.write('HTTP/1.1 200 Connection Established' + '\r\n' +
-      'Proxy-agent: Node-Proxy' + '\r\n' +
+  socket.write('HTTP/1.1 200 Connection Established\r\n' +
+      'Proxy-agent: Node-Proxy\r\n' +
```

Comment:
It doesn't report multiline concatenations because people often use those intentionally (e.g. to avoid long lines). 

Related: https://github.com/eslint/eslint/issues/7947